src/rules/objectLiteralSortKeysRule.ts

```diff
@@ -62,7 +62,7 @@ class ObjectLiteralSortKeysWalker extends Lint.RuleWalker {
         if (sortedState) {
             const lastSortedKey = this.lastSortedKeyStack[this.lastSortedKeyStack.length - 1];
             const keyNode = node.name;
-            if (keyNode.kind === ts.SyntaxKind.Identifier) {
+            if (keyNode.kind === ts.SyntaxKind.Identifier || keyNode.kind === ts.SyntaxKind.StringLiteral) {
                 const key = (<ts.Identifier> keyNode).text;
```

Comment:
this type assertion is no longer accurate with the change in the line above. I suggest refactoring into a type guard

```ts
function isIdentifierOrStringLiteral(node: ts.Node): node is (ts.Identifier | ts.StringLiteral) {
    return node.kind === ts.SyntaxKind.Identifier || node.kind === ts.SyntaxKind.StringLiteral;
}
```